src/icons/navigator/Invite.tsx

```diff
@@ -37,4 +41,8 @@ export function Invite() {
   )
 }
 
+Invite.defaultProps = {
+  color: '#B4B9BD',
```

Comment:
This isn't one of our standard colors right? Are there other usages of this that needs this default? If not, I'd vote for just using gray3 as the default, since the navigator already overrides it with this non standard color.